source/Engine/ResourceTypes/ModelFormats/HatchModel.cpp

```diff
@@ -269,22 +269,25 @@ Mesh* HatchModel::ReadMesh(IModel* model, Stream* stream) {
 	}
 
 	Uint32 vertexCount = stream->ReadUInt32();
-	if (!vertexCount) {
-		Log::Print(Log::LOG_ERROR, "Mesh \"%s\" has no vertices!", mesh->Name);
-		return nullptr;
-	}
```

Comment:
We want this to always be validated, to prevent Release builds from trying to load malformed models.